src/webroot/js/cake-core.js

```diff
@@ -0,0 +1,23 @@
+//$(document).ready(function () {
+var elements = document.getElementsByClassName('cake-core-postLink');
+
+var postLinkFunction = function (event) {
+    var formName = this.getAttribute("data-cake-core-form");
+    var confirmMessage = this.getAttribute('data-cake-core-confirm');
+    if (confirmMessage !== null && confirmMessage !== 'undefined') {
+        if (confirm(confirmMessage)) {
+            document[formName].submit();
+            return false;
+        }
+        return true;
```

Comment:
I have to be honest here. I don't really know if it is correct to return true here. Does someone know if this is correct?